pkg/jobrunaggregator/jobruninstallanalyzer/cmd.go

```diff
@@ -0,0 +1,152 @@
+package jobruninstallanalyzer
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"k8s.io/apimachinery/pkg/util/clock"
+
+	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorlib"
+)
+
+type JobRunsInstallAnalyzerFlags struct {
```

Comment:
to work with /payload commands, you'll need `--aggregation-id` and `--explicit-gcs-prefix`